pallets/protos/src/lib.rs

```diff
@@ -845,25 +867,6 @@ pub mod pallet {
 			let found: Vec<_> = categories
 				.into_iter()
 				.filter(|cat| match cat {
-					Categories::Trait(proto_shard_trait) => {
```

Comment:
Though, this removes completely the filter by trait name or id which is a feature that I understood was needed, no?

Reply:
Also mentioned here https://github.com/fragcolor-xyz/asset-store/issues/37

Reply:
So maybe we could leave that .filter by Categories::Trait and use your new structure `<Traits<T>>::contains_key(&trait_id)`

Reply:
but name is not available anymore, you can pass ID still no?

Reply:
basically now we would pass right IDs

Reply:
Oh yes I see, just by matching the objects will work now since there is only one field. Ok!